packages/lesswrong/components/posts/TableOfContents/TableOfContentsRow.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { PureComponent, Component } from 'react';
+import { Components, registerComponent } from 'meteor/vulcan:core';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import classNames from 'classnames';
+
+const styles = theme => ({
+  root: {
+    ...theme.typography.body2,
+    ...theme.typography.postStyle,
+  },
+
+  // For the highlighted section only, disable the half-opacity-on-hover effect
+  // that's otherwise globally applied to <a> tags.
+  highlighted: {
+    fontWeight: "bold",
+    '& $link': {
+      color: "black !important",
```

Comment:
I am quite sure this isn't necessary. Though you might have to increase specificity.